docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -1571,9 +1568,12 @@ Example in a command:
 
 ### Task Fields Override Hierarchy
 
-Some fields can be specified at multiple levels in the YAML. The task's specific
-fields will be taken into priority in the following order (from highest to
-lowest):
+Some task fields can be specified at multiple levels in the YAML. These fields
+are:
```

Comment:
Oh yeah, I actually planned to remove this and forgot. I was hoping to just list out all the task fields that you can define at multiple levels and state the precedence rules for all of them in one nice clean list. Unfortunately, it became too complicated to explain which keys can be used where, because it turns out some task fields can be defined in some places but not others (e.g. `stepback` can be defined at the global level, but `patchable` can't).

This should be removed now.